.goreleaser.yml

```diff
@@ -47,6 +46,28 @@ builds:
       - darwin
     # hooks:
     #   post: gon -log-level debug ci/gon.hcl
+
+  - id: space-win
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w -X main.ipfsaddr={{ .Env.IPFS_ADDR }}
+      - -X main.mongousr={{ .Env.MONGO_USR }}
+      - -X main.mongopw={{ .Env.MONGO_PW }}
+      - -X main.mongohost={{ .Env.MONGO_HOST }}
+      - -X main.mongorepset={{ .Env.MONGO_REPLICA_SET }}
+      - -X main.spaceapi={{ .Env.SERVICES_API_URL }}
+      - -X main.spacehubauth={{ .Env.SERVICES_HUB_AUTH_URL }}
+      - -X main.textilehub={{ .Env.TXL_HUB_TARGET }}
+      - -X main.textilehubma={{ .Env.TXL_HUB_MA }}
+      - -X main.textilethreads={{ .Env.TXL_THREADS_TARGET }}
+    main: ./cmd/space-daemon/main.go
+    binary: space
+    goos:
+      - windows
+    ignore:
+      - goos: windows
```

Comment:
@perfectmak @dmerrill6 I noticed in earlier versions 386 was disabled for windows. Do you guys know the reason for that? Kept it disabled in case it breaks the build but and we can enable it again and for now just enabled x86 on windows.

Reply:
Actually, I tested it locally and the goreleaser build successfully for both architectures so reenabled it.